sourcecode/hub/resources/views/content/standalone-view.blade.php

```diff
@@ -1,71 +1,64 @@
-                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="{{ trans('messages.close')}}"></button>
                 </div>
                 <div class="modal-body border-0">
                     <p>{{ trans('messages.this-game-has-content')}}</p>
                     <div class="d-flex flex-wrap gap-2 mt-4">
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Copyrighted">{{ trans('messages.copyrighted')}}</button>
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Violent">{{ trans('messages.violent')}}</button>
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Sexual">{{ trans('messages.sexual')}}</button>
-                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="Other">{{ trans('messages.other')}}</button>
+                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="{{ trans('messages.copyright')}}">{{ trans('messages.copyright')}}</button>
+                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="{{ trans('messages.violence')}}">{{ trans('messages.violence')}}</button>
+                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="{{ trans('messages.sexual')}}">{{ trans('messages.sexual')}}</button>
+                        <button class="btn btn-secondary btn-sm p-5 m-1" type="button" aria-label="{{ trans('messages.other')}}">{{ trans('messages.other')}}</button>
                     </div>
                     <div class="mt-4">
-                        <textarea class="form-control" placeholder="Describe the problem here..." aria-label="Problem Description"></textarea>
+                        <textarea class="form-control" placeholder="{{ trans('messages.describe-the-problem-here')}}" aria-label="{{ trans('messages.problem-description')}}"></textarea>
                     </div>
                 </div>
                 <div class="modal-footer border-0 d-flex justify-content-center align-items-center py-4">
-                    <button type="button" class="btn btn-primary mx-2" data-bs-toggle="modal" data-bs-target="#reportSuccessModal" aria-label="Report Content"> {{ trans('messages.report-content')}}</button>
-                    <button type="button" class="btn btn-secondary mx-2" data-bs-dismiss="modal" aria-label="Cancel"> {{ trans('messages.cancel')}}</button>
+                    <button type="button" class="btn btn-primary mx-2" data-bs-toggle="modal" data-bs-target="#reportSuccessModal" aria-label="{{ trans('messages.report-content')}}"> {{ trans('messages.report-content')}}</button>
```

Comment:
The content of the element is a readable text, no need to repeat it using `aria-label`